### ISSUE TITLE
F-Droid can't build

### ISSUE THREAD

**Issue by licaon-kter** [agateau/burgerparty#24]
...as there's no APK in https://github.com/agateau/burgerparty/releases/tag/1.4.4

ref: https://gitlab.com/fdroid/fdroiddata/-/jobs/7242009038#L762

**Comment by agateau** [comment]
Hi, it should be good now, but I can't find any new build on GitLab. Can you verify?

**Comment by licaon-kter** [comment]
It will be retried on the main server, see http://monitor.f-droid.org

The Gitlab CI is just for checking builds ;)

**Comment by licaon-kter** [comment]
https://monitor.f-droid.org/builds/log/com.agateau.burgerparty/34#site-footer

**Comment by licaon-kter** [comment]
No APK for https://github.com/agateau/burgerparty/releases/tag/1.4.5 ?

**Comment by agateau** [comment]
> No APK for https://github.com/agateau/burgerparty/releases/tag/1.4.5 ?

Yes there is one: https://github.com/agateau/burgerparty/releases/download/1.4.5/burgerparty-1.4.5-agc.apk

**Comment by licaon-kter** [comment]
so this will be fixed in the next version?

**Comment by linsui** [comment]
Remove DependencyInfoBlock for F-Droid

We find that there is a `DependencyInfoBlock` in your APK.

It's a [Signing block](https://source.android.com/docs/security/features/apksigning/v2#apk-signing-block) added by AGP and encrypted with the Google public key so it can't be read by anyone else except Google. You can read more about it [here](https://gitlab.com/fdroid/admin/-/issues/367), [here](https://gitlab.com/fdroid/fdroidserver/-/issues/1056) and [here](https://android.izzysoft.de/articles/named/iod-scan-apkchecks?lang=en#blobs).

While this was added a while ago, we were only enforcing it for new apps, and recently we started scanning updates too.

Could you please disable it with the following code?

```
android {
    dependenciesInfo {
        // Disables dependency metadata when building APKs.
        includeInApk = false
        // Disables dependency metadata when building Android App Bundles.
        includeInBundle = false
    }
}
```

Thanks!
